app.js

```diff
@@ -1 +1,94 @@
-'use strict';
\ No newline at end of file
+'use strict';
+
+Product.allProducts = [];
```

Comment:
Since this is a property of `Product` this line should probably be after the declaration of `Product`. Your code will work without doing this, but it is more organized like this.